maven/src/main/resources/java-source.ftlh

```diff
@@ -167,17 +170,19 @@ public class ${java_classname} implements Serializable {
   <#list entity.attributes as attribute>
     <#if attribute.wildcard>
     @SuppressWarnings("unchecked")
-    private final AttributeDescriptor<${attribute.type}> ${attribute.name}Descriptor =
-      (AttributeDescriptor) descriptor.getAttribute("${attribute.name}.*");
+    private final Wildcard<${attribute.type}> ${attribute.name}Descriptor =
+      EntityAwareAttributeDescriptor.wildcard(descriptor, (AttributeDescriptor) descriptor.getAttribute("${attribute.name}.*"));
```

Comment:
We know the types so casting to `(AttributeDescriptor)` can be imho removed and also `@SuppressWarnings("unchecked")` right?

Reply:
:+1: